blog.html

```diff
@@ -22,6 +22,125 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_370_release_announcement"></a>
+                    <a href="#apache_kafka_370_release_announcement">Apache Kafka 3.7.0 Release Announcement</a>
+                </h2>
+                February 2024 - Stanislav Kozlovski (<a href="https://twitter.com/BdKozlovski">@BdKozlovski</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.7.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.7.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_7_0">Upgrading to 3.7.0 from any version 0.8.x through 3.6.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
```

Comment:
I think this link should be `https://kafka.apache.org/documentation.html#upgrade_3_7_0`. But as I said in the vote thread, this section is missing from the 3.7 docs.

Reply:
For this, no need for another RC, just open a PR against kafka-site that adds this section. The section for 3.7.0 can probably be pretty much the same as for 3.6.0 as I don't think there's anything specific in 3.7.

Reply:
good point!